siap/siap_client/utils.py

```diff
@@ -330,6 +333,42 @@ def get_or_create_programme(
     return programme
 
 
+def create_subventions(subventions, financement, lot):
+    for subvention in subventions:
+        if subvention["type"] == financement:
+            Pret.objects.create(lot=lot, montant=subvention["montant"], autre="")
+
+
+def create_prets(prets, lot):
+    for pret in prets:
+        Pret.objects.create(lot=lot, montant=pret["montant"], autre="")
+
+
+def create_financements(operation, financement, lot):
+    if "plansFinancement" in operation and operation["plansFinancement"]:
```

Comment:
```suggestion
    if operation.get("plansFinancement"):
```
Je pense que ça fait la même chose !